PendingReleaseNotes.md

```diff
@@ -15,3 +15,5 @@ v1.7...
 ### Core
 
 ### Ceph
+
+* Stop standby mds agents before upgrading ceph filesystem
```

Comment:
Actually please revert the changes to pending release notes since we want to backport this to 1.6. This is only required for v1.7 features.